### PR TITLE
feat: Add SSG/One-Chunk CPU project.

### PR DESCRIPTION
**Pull request by xxMattewxx**
This PR adds support for the [https://github.com/minecrafthome/OneChunk](SSG/One-Chunk) project, which aims to find all world seeds with an 11-eye portal in MC versions 1.7 and 1.8.
Before running taskgen, please make sure all the new binaries have been set +x flag and the daemons are running correctly.

### CODE REVIEW

**Comment by ItzN00bPvP** [comment]
a preventive:
/fix
I don't trust matts code thats bug free

**Comment by Hyenadae** [comment]
At least in testing we made good progress and (for now) we can create the tasks we need, run the application on the Raspberry Pi 3/4 64bit and upload our results. I don't see anything broken or wrong (yet)

File: images/makeproject/project/bin/onechunk_assimilator.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+                    if canonical_result == None:
+                            return
+
+                    path = boinc_project_path.project_path("onechunk_results")
+                    input_path = self.get_file_path(canonical_result)
+
+                    with open(input_path) as input_file:
+                        input_str = input_file.read()
+
+                    try:
```
Comment (hube12) [inline images/makeproject/project/bin/onechunk_assimilator.py:21]: can be simplified to `os.makedirs(exist_ok=True)`

**Review by Hyenadae — DISMISSED** [review]

**Review by hube12 — APPROVED** [review]
This seems fine to me, the python side seems safe enough

File: images/makeproject/project/onechunk_workgen.sh [hunk 2]
```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+IFS=' '
+filename='md5hashes.txt'
```
Comment (hube12) [inline images/makeproject/project/onechunk_workgen.sh:3]: Where is that file?

Comment (xxMattewxx) [reply images/makeproject/project/onechunk_workgen.sh]: Added it and changed the file name as requested.

**Review by hube12 — APPROVED** [review]

File: images/makeproject/project/onechunk_workgen.sh [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    read -a strarr <<< "$line"
+    ./bin/create_work --appname onechunk \
+        --result_template templates/onechunk_out \
+        --remote_file "https://ssg.seedfinding.com/tasks/get?taskid=${strarr[0]}"\
+        "${strarr[1]}" \
+        "${strarr[2]}" \
+        --priority 12000 \
+        --min_quorum 2 \
+        --credit 2500 \
+        --delay_bound 259200
```
Comment (BoySanic) [inline images/makeproject/project/onechunk_workgen.sh:15]: Can we increase this to 1209600? 

I'm worried 3 days won't be enough for ARM devices that are shared across multiple projects.

2 weeks should be plenty and sounds about right for some of the other CPU projects I've seen on BOINC.
```suggestion
        --delay_bound 1209600
```

**Review by BoySanic — CHANGES_REQUESTED** [review]
I was also thinking maybe we can change tasks.txt to like, onechunk_tasks.txt to be a little more descriptive.

File: images/makeproject/project/templates/onechunk_in [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+    <file_info>
+    </file_info>
+    <workunit>
+        <file_ref>
+            <open_name>jf_MD5</open_name>
+            <copy_file/>
+        </file_ref>
+        <rsc_fpops_est>2.8e13</rsc_fpops_est>
+        <rsc_fpops_bound>2.8e20</rsc_fpops_bound>
+        <delay_bound>259200</delay_bound>
```
Comment (BoySanic) [inline images/makeproject/project/templates/onechunk_in:11]: Same thing with the delay_bound. Though it's redundant to put it both in the command and in the template.
```suggestion
        <delay_bound>1209600</delay_bound>
```

**Review by BoySanic — APPROVED** [review]
LGTM! No additional requests for changes here.

:shipit:

**Review by Hyenadae — APPROVED** [review]

**Pull request merged**